sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -94,56 +126,47 @@ export interface ListTagsOptions extends OperationOptions {
 }
 
 // @public
-export type RegistryArtifactOrderBy = "timedesc" | "timeasc";
+export type ManifestOrderBy = "timeDesc" | "timeAsc";
 
 // @public
-export interface RegistryArtifactProperties {
-    cpuArchitecture?: string;
-    createdOn?: Date;
-    digest?: string;
-    lastUpdatedOn?: Date;
-    operatingSystem?: string;
-    registryArtifacts?: RegistryArtifactProperties[];
-    repository?: string;
-    size?: number;
-    tags?: string[];
-    writeableProperties?: ContentProperties;
+export class RegistryArtifact {
+    // @internal
+    constructor(registryUrl: string, repositoryName: string, tagOrDigest: string, client: GeneratedClient);
```

Comment:
same thinking, let's make this an interface since it's not a `Client`

Reply:
Fixed.